baseclasses/pyAero_problem.py

```diff
@@ -948,6 +1040,96 @@ def _getDVSens(self, func):
 
         return rDict
 
+    def evalDVsSensFwd(self, xDvDot):
+        """
+        fwd mod deriv of setDesignVars
+        """
+        aeroDvsDot = {}
+        bcDvsDot = defaultdict(lambda: {})
+        actDvsDot = defaultdict(lambda: {})
+
+        for dvName in self.DVs:
+            if dvName in xDvDot:
+                dv = self.DVs[dvName]
+                key = dv.key
+                family = dv.family
+                value = xDvDot[dvName]
+
+                if key in self.possibleBCDVs:
+                    bcDvsDot[family][key] = value
+
+                elif key in self.possibleActuatorDVs:
+                    actDvsDot[family][key] = value
+                else:
+                    aeroDvsDot[key] = value
+
+        return aeroDvsDot, bcDvsDot, actDvsDot
+
+    def evalDVsSensBwd(self, aeroDvBar, BCDataBar, actDataBar):
+        """This internal furncion is used to convert the raw array output from
```

Comment:
Two dumb questions here:
- Why the fwd and bwd function differ (i.e. why we have a different treatment for mach and altitude only for the bwd sens)?
- how are these functions going to be used, and how are we currently dealing with the sens raw data?